public/styles/index.css

```diff
@@ -37,6 +37,9 @@
         margin-top: 32%;
         margin-left: 32%;
     }
+    .__shecodeheader_subtitle h4 {
+        font-size: 25px!important;
+    }
```

Comment:
Another `h4` with a different font size and an `!important`.

If there is a base style that needs to be overwritten, I think it may be possible to do that in one place